src/Detail/components/BookInfoBox/index.tsx

```diff
@@ -38,8 +67,20 @@ function BookInfoBox({
             </S.BookInfoHeaderItem>
           </S.BookInfoHeaderItemWrapper>
         </S.BookInfoHeader>
-        <S.BookInfoTitle backgroundColor={bookBackgroundColor}>
-          {title}
+        <S.BookInfoTitle>
+          <S.BookInfoTitleText backgroundColor={bookBackgroundColor}>
+            {title}
+          </S.BookInfoTitleText>
+          {isLogin &&
+            (isFavorite ? (
+              <S.FavoriteBtn type="button" onClick={handleFavoriteBtn}>
+                <IcZigzagStarOn />
+              </S.FavoriteBtn>
+            ) : (
+              <S.FavoriteBtn type="button" onClick={handleFavoriteBtn}>
+                <IcZigzagStarOff />
+              </S.FavoriteBtn>
+            ))}
```

Comment:
P2: 요렇게 표현하면 중복코드를 줄일 수 있을 것 같아요 !
```suggestion
          {isLogin && (
              <S.FavoriteBtn type="button" onClick={handleFavoriteBtn}>
               { isFavorite ? <IcZigzagStarOn /> : <IcZigzagStarOff />}
              </S.FavoriteBtn>
            )}
```